cw/cw305/simple_capture_traces.py

```diff
@@ -61,9 +61,6 @@ def run_capture(capture_cfg, ot, ktp):
     if not ret:
       print('Failed capture')
       continue
-    # This value may need to be updated if the trace dB factor changes.
-    elif min(ret.wave) < -0.45:
-      continue
```

Comment:
I also removed this part in #23 but I will drop that commit since this PR is the right place for this change.

Reply:
Thanks.